Examples/MAX32655/ADC/README.md

```diff
@@ -19,8 +19,9 @@ Universal instructions on building, flashing, and debugging this project can be
 
 ## Setup
 
-##### Board Selection
-Before building firmware you must select the correct value for _BOARD_  in "project.mk", either "EvKit\_V1" or "FTHR\_Apps\_P1", depending on the EV kit you are using to run the example.
+### Project Usage
```

Comment:
You've duplicated the Project Usage section here for the ADC README.  The one marked in red should be removed.

<img width="437" alt="image" src="https://github.com/Analog-Devices-MSDK/msdk/assets/38844790/1473fbd3-ba99-4648-85a9-ca69b40e933a">